### PR TITLE
Cart quantity closes #21

### LINKED ISSUE #21: Adjusting Quantity of an Item in Cart
Background: My cart has an item in it
      As a visitor
      When I visit "/cart"
      Then I should see my item with a quantity of 1
      And when I increase the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the increase
      And the subtotal for that item should increase
      And the total for the cart should match that increase
      And when I decrease the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the decrease
      And the subtotal for that item should decrease
      And the total for the cart should match that decrease

### PR DESCRIPTION
**Pull request by Dpalazzari**
@Laszlo-JFLMTCO @drod1000 @njgheorghita 

This is what Nick and I worked on today. We added the ability to add/subtract items from directly within the cart.

One problem remains: The subtraction method keeps going into the negatives if you keep clicking the "Subtract button".

Also, if the '/cart' routes are causing problems, we may or may not have added to that problem with this PR.

### CODE REVIEW

File: app/views/carts/index.html.erb [hunk 1]
```diff
@@ -16,12 +16,16 @@
         <td><%= content.description %></td>
         <td><img src="<%= content.image_url %>" class="img-responsive" id="outing-photo"></td>
         <td><%= content.adjusted_cost_currency %></td>
-        <td><%= content_qty %></td>
+        <td id='quantity'><%= link_to "+", increment_path(outing_id: content.id),method: :POST, class:'btn btn-success btn-xs', id: 'add' %>
+          <%= content_qty %>
+          <% if content_qty >= 2 %>
+            <%= link_to "-", decrement_path(outing_id: content.id), method: :POST, class:'btn btn-warning btn-xs'%></td></td>
+          <% end %>
```
Comment (njgheorghita) [inline app/views/carts/index.html.erb:23]: nice idea @drod1000

**Comment by njgheorghita** [comment]
i paired on this code, looks good to be merged

**Comment by drod1000** [comment]
@Dpalazzari Looks good to me. Great job. closes #32

**Comment by Laszlo-JFLMTCO** [comment]
Looking good based on our joint review! Merge it!

**Pull request merged**